Core/ClientSMLSWIG/Python/SConscript

```diff
@@ -17,14 +17,20 @@ python_sml_alias = clone['SML_PYTHON_ALIAS']
 
 inc_path = sysconfig.get_path('include')
 if os.name == 'nt':
-    lib_path = os.path.join(sysconfig.get_config_vars('BINDIR')[0], 'libs')
+    lib_path = [
+        os.path.join(sysconfig.get_config_vars('BINDIR')[0], 'libs'),
+        os.path.join(sysconfig.get_config_vars('LIBDEST')[0], '..\\libs'),
+    ]
     pylib = 'python' + sysconfig.get_config_vars('VERSION')[0]
 else:
     lib_path = sysconfig.get_config_vars('LIBDIR')[0]
     pylib = sysconfig.get_config_vars('LIBRARY')[0]
 lib_install_dir = clone['OUT_DIR']
 
-clone.Append(CPPPATH = inc_path, LIBPATH = lib_path, LIBS = pylib)
+clone.Append(CPPPATH = inc_path)
+
+if (not env['ENSCONS_ACTIVE']) or os.name == 'nt':
+    clone.Append(LIBPATH = lib_path, LIBS = pylib)
```

Comment:
On linux and macos, this specifies that the linker should look for the `python3X` library, which don't exist on the manylinux/macos images.

The reasons for this are complicated, but basically, "its probably not what you want" (and they're right), but having these here prevents them being added for linking.

(The "not enscons_active" part is to ensure cibuildwheel-less builds still "work" on whatever platform they're designed to build on)

Reply:
The most direct comment and traceback i could find about this would be here: https://github.com/pypa/manylinux/issues/191#issuecomment-386489125

Adding in the libraries to be statically compiled against (which is what scons is trying to do here) then just results in extra sizes of the binary, and the shipping of those binaries in `manylinux` docker files would increase the container size.

I'd say that this is the right practical decision, even if it kinda makes life harder with the way scons tries to build this by default.

Reply:
Makes sense! Could you add a comment?